lib/decorateWithMetrics.js

```diff
@@ -20,7 +25,7 @@ function decorateWithMetrics(func, alias = func.name) {
     stats[alias] = [];
 
     function decorated(...args) {
-        const startTime = performance.now();
+        const startTime = performance.now() - APP_LAUNCH_TIME;
```

Comment:
I'm trying to understand the reliability of this next usage of `performance.now()` here and which metric this will make more reliable. Is it just the `endTime`? Since all the other metrics are relative to `performance.now()` but that one will tell us the actual time from when Onyx started doing work and when it finished for each method?

Reply:
This doesn't make anything more reliable, just keeps our start/end times relative to app launch 

Differences between `performance.now()` are reliable but the initial time given by `performance.now()` is ridiculously high - and I can't match the value to anything - it's not the current unix ts, it's not the time since app launch
It's a value like 508 million and increases as time passes. Might be relative to midnight 00:00 UTC ? But that doesn't help us track boot time 😄 

On the old flow, when this was provided by Chrome, the value will be very close to `0` initially and so you know a value of `1000` means something started/ended a second past app launch

By capturing our own reference point APP_LAUNCH_TIME we keep start/end times relative to that point
so we have more useful information

Reply:
If I remove this, I'll get this ATM: "Last call finished at: 8907.4min" so it's not midnight
It might be the time since last reboot, let me check
Yep seems to be it: "Last call finished at: 10.0min"
I guess it uses some clock managed by the OS and that's what it returns

Reply:
Haha, ok awesome, thanks for the explanation. 
Very weird, but now that we are interested in something besides relative times also very necessary.